### ISSUE TITLE
Jason Grad Poster

### ISSUE THREAD

**Issue by balacij** [JacquesCarette/Drasil#2963]
I've [posted a first draft of my poster](https://github.com/JacquesCarette/Drasil/blob/jasonWriting/People/Jason/poster/poster.pdf) that I'll be presenting for Tuesday April 19th's graduate poster session. I'm still actively working on it (trying to shorten sentences at least, so that I can add more relevant/focused keywords).

If anyone has any free time to give it a quick look, I would greatly appreciate it, but, of course, I understand if not possible given the time crunch and it being a bit late.

Thank you to @peter-michalski, whose recent questions about my thesis helped me focus some ideas that made it into my poster.

(The poster is intended to be 36" x 24", the font is 20pt at the moment.)

**Comment by smiths** [comment]
@balacij, you definitely have a challenge on your hands to express the goals of your research.  Explaining Drasil is hard enough, and you have the extra problem of explaining Drasil AND how you are improving Drasil.  In the space available I think you did a great job, especially with your two figures.

It is too bad that there isn't more room for examples.  I've found that people more readily understand examples.  Hopefully you are using examples in your thesis write-up.  :smile:

I noticed a few minor things.  You may have already noted these in your own revisions:

- In the introductory paragraph you can replace "as described in [4]" replace with just "[4]".  You actually can replace the entire mention of Smith and Lai, since the reference [4] has this information.
- When introducing the website at the end of the first paragraph, the semicolon should be changed to a full colon
- What about putting a cloud around "Mathematical Knowledge" in the second Figure, just like the first, but in the second figure, maybe changing the colour to green?  The consistency of the cloud would show the reader that it is the same knowledge, but now improved.
- The last sentence of the Conclusion is in a smaller font.

Good luck with the poster session!

**Comment by balacij** [comment]
Thank you very much @smiths! I definitely wish I could have had a few examples, I wasn't able to fit them here, but I will make sure to have them on my thesis. Unfortunately, I've already submitted the poster now, I didn't see this message in time, but I might be able to marker them in just before the session. However, I did fix the last sentence of the Conclusion by moving it to the spoken portion of the poster. A green cloud around the "Mathematical Knowledge" in the second figure would have been a nice change, I'll at least make sure to add this in my thesis now.

Thank you again! I'll go ahead and close this ticket now, but if there's any other feedback from anyone, I would still appreciate it for the future.
